12-kubernetes-cluster.tf

```diff
@@ -40,7 +40,8 @@ resource "azurerm_kubernetes_cluster" "kubernetes_cluster" {
     var.service_shortname
   )
 
-  oidc_issuer_enabled = true
+  oidc_issuer_enabled       = true
+  workload_identity_enabled = true
```

Comment:
this probably shouldn't be unconditionally enabled as I think it requires a preview feature flag to be enabled first, but you could do this if you are going to run the command to enable on all subs first